contrib/podmanimage/machine/Containerfile

```diff
@@ -0,0 +1,26 @@
+FROM quay.io/centos-bootc/fedora-bootc:eln
+RUN dnf -y install podman subscription-manager crun crun-wasm crun-krun chrony; dnf -y clean all
+RUN dnf -y install skopeo buildah; dnf -y clean all
+RUN systemctl enable podman.service
+RUN printf "\nconfdir /etc/chrony.d\n" >> /etc/chrony.conf
+
+COPY 10-inotify-instances.conf /usr/etc/sysctl.d/10-inotify-instances.conf
+COPY 50-podman-makestep.conf /usr/etc/chrony.d/50-podman-makestep.conf
+COPY 999-podman-machine.conf /usr/etc/containers/registires.conf.d/999-podman-machine.conf
+COPY containers-etc.conf /usr/etc/containers/containers.conf
+COPY delegate.conf /usr/etc/systemd/system/user@.service.d/delegate.conf
+COPY docker-host.sh /usr/etc/profile.d/docker-host.sh
+COPY podman-docker.conf /usr/etc/tmpfiles.d/podman-docker.conf
+COPY subuid /usr/etc/subgid
+COPY subuid /usr/etc/subuid
+RUN groupadd -g 501 core; useradd -u 501 -g 501 core
```

Comment:
This one is tricky...I know we're adding it for compatibility, but going forward I think it'd probably make sense to allocate a `podman` user for example.

But even more importantly it will work better if we change things to allocate the user on firstboot today.

More on this in https://centos.github.io/centos-bootc/builds/#injecting-users-at-build-time (that could be expanded too)

Reply:
Also, why 501 versus the default?

Reply:
Just going by the entries in ignition.go.

Reply:
Looks like the default user on a MAC is UID=501